awx/ui/client/src/access/rbac-role-column/roleList.directive.js

```diff
@@ -21,9 +21,22 @@ export default
                         }))
                         .concat(scope.deleteTarget.summary_fields
                             .indirect_access.map((i) => {
-                                i.role.explicit = false;
-                                return i.role;
+                                if (i.descendant_roles.includes('admin_role')) {
```

Comment:
why only for admin roles?

Reply:
the indirect_access shows a full role tree, so if you inherit admin_role, it shows *all* roles. We want to shortcut this rather than show all roles if you inherit admin.

Reply:
so, "if admin is in the indirect role list, show only admin, otherwise show all indirect roles"